example/main.go

```diff
@@ -106,7 +106,9 @@ func ScheduledJob3(ctx context.Context) {
 }
 
 func main() {
-	Pool = worker.NewWorkerPool(4, 4)
+	Pool = worker.NewWorkerPool(
```

Comment:
it breaks backward code compatibility. We only have two variables here i think we don't need to add config.go it's simple library